src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCaller.java

```diff
@@ -197,7 +197,7 @@ public AssemblyRegionEvaluator assemblyRegionEvaluator() {
     @Override
     public void onTraversalStart() {
         final ReferenceSequenceFile referenceReader = getReferenceReader(referenceArguments);
-        hcEngine = new HaplotypeCallerEngine(hcArgs, getHeaderForReads(), referenceReader);
+        hcEngine = new HaplotypeCallerEngine(hcArgs, createOutputBamIndex, createOutputBamMD5, getHeaderForReads(), referenceReader);
```

Comment:
Yeah - I considered a wrapper but it didn't seem worth it. I also thought about having the tool(s) propagate these values to HaplotypeCallerArgs/(M2ArgumentCollection) before initializing the engines. Not sure if its worth it, they'd still have to be threaded through the lower levels.